documentation/MIGRATION guide for bumping up to v30.0.0.md

```diff
@@ -0,0 +1,196 @@
+```typescript
+const result = client.availabilitySets.list(this.resourceName);
+for await (const item of result.byPage({maxPageSize:2})) {
+    handle(item);
+}
+```
+Refer to [@azure/core-paging](https://www.npmjs.com/package/@azure/core-paging) for more details.
+
+## Long Running Operations
+
+Javascript/Typescript SDK provide two operations with the same signature for Long Running Operations for our customers. One operation returns a LROPoller whose name starts with `begin` prefix, the other will poll until finished and return the final result to our customer that shares the same name with the rest api operationId.  
+In version 30.0.0+, we keep this feature except 
+1. we change those operations names with a `begin` prefix and `AndWait` suffix which will poll until finshed and return the result directly.  
+1. we change the poller type of the response
+
+<!-- markdownlint-disable MD033 -->
+<table>
+  <tr>
+    <td colspan="2">
+      <p>
+        Operations that returns a poller.
+      </p>
+    </td>
+  </tr>
+  <tr>
+    <td>
+      <pre lang="typescript">
+  beginCreateOrUpdate(
+    resourceGroupName: string,
+    cloudServiceName: string,
+    options?: Models.CloudServicesBeginCreateOrUpdateOptionalParams
+  ): Promise<
+       msRestAzure.LROPoller
+     >
+      </pre>
+    </td>
+    <td>
+      <pre lang="typescript">
+  async beginCreateOrUpdate(
+    resourceGroupName: string,
+    cloudServiceName: string,
+    options?: CloudServicesCreateOrUpdateOptionalParams
+  ): Promise<
+    PollerLike<
+      PollOperationState<
+        CloudServicesCreateOrUpdateResponse
+      >,
+      CloudServicesCreateOrUpdateResponse
+    >
+  >
+      </pre>
+    </td>
+  </tr>
+  <tr>
+    <td colspan="2">
+      <p>
+        Operations that will poll until finish and return the result directly
+      </p>
+    </td>
+  </tr>
+  <tr>
+    <td>
+      <pre lang="typescript">
+  createOrUpdate(
+    resourceGroupName: string,
+    cloudServiceName: string,
+    options?: Models.CloudServicesCreateOrUpdateOptionalParams
+  ): Promise<
+       Models.CloudServicesCreateOrUpdateResponse
+     >
+      </pre>
+    </td>
+    <td>
+      <pre lang="typescript">
+  async beginCreateOrUpdateAndWait(
+    resourceGroupName: string,
+    cloudServiceName: string,
+    options?: CloudServicesCreateOrUpdateOptionalParams
+  ): Promise<
+       CloudServicesCreateOrUpdateResponse
+     >
+      </pre>
+    </td>
+  </tr>
+</table>
+<!-- markdownlint-enable MD033 -->
+
+**There are also some changes inside the Poller's usages. In general,**
+1. we change `pollUntilFinished` into `pollUntilDone` and change `isFinished` into `isDone`.
+1. We add `isStopped`, `stopPolling`, `cancelOperation`, `onProgress` to give users a better control of the Long Running Operation. 
+
+Refer to [@azure/core-lro](https://www.npmjs.com/package/@azure/core-lro) for more details.
```

Comment:
Add a Need help section like Python so people can open an issue

https://github.com/Azure/azure-sdk-for-python/blob/main/doc/sphinx/python_mgmt_migration_guide.rst

Reply:
I add `If you have encountered an issue during migration, please file an issue via Github Issues <https://github.com/Azure/azure-sdk-for-js/issues>and make sure you add the "Preview" label to the issue` but found there's no such a label ? which label do you think we should put in here ? I will change it accordingly.